add nuance via websockets

patbit opened the pull request.

patbit committed:
add nuance via websockets

russaa reviewed (changes requested):
see remarks for bing-plugin

patbit committed:
improved versions of nuance/bing asr via ws

removed unnessesary code
add JSDoc
disable log messages
rename folder and id -> all lowcase
